test/functional/cmdLineTests/gcRegressionTests/gcRegressionTests.xml

```diff
@@ -227,6 +227,13 @@
  	<output regex="no" type="failure">Test ran to completion</output>
  	<output regex="no" type="success">java.lang.OutOfMemoryError</output>
  </test>
+ <test id="Excessive GC throws OOM on RISC-V">
+ 	<echo value="CMVC 148977:  Excessive GC throws OOM may fail if there is other activity on the machine - at least 5 GCs are required, in the 20 second test" />
+ 	<!-- does the test fail to run (thus implying that we encountered excessive GC)? -->
+ 	<command>$EXE$ $XINT$ $ARGS_FOR_ALL_TESTS$ -Xdump:none -Xms2m -Xmx2m -Xgc:fvtest=forceExcessiveAllocFailureAfter=5 -verbose:gc -Xverbosegclog:foo.log $CP$ com.ibm.tests.garbagecollector.SpinAllocate 20</command>
```

Comment:
You did increase parameter range from [1-60] to [1-200] for `SpinAllocate`. Is there an intention to increase it here?

Reply:
Yes, we have to increase the value there to ensure it passes the test case correctly as this is pretty much the same timeout issue with `TestGCPolicyNogc`.

Reply:
Are you going to change timeout parameter in this commit?

Reply:
There is no need to do so based on the test results as we already increase the threshold

Reply:
Removed the change on threshold as unnecessary given the initial memory setting is modified to fit the case on RISC-V.